packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -181,6 +181,22 @@ def get_ps_output():
         assert len(get_ps_output().split()) <= 4, 'UCR app has more than 4 processes running in its pid namespace'
 
 
+def test_memory_profiling(dcos_api_session):
+    # Test that we can fetch raw memory profiles
```

Comment:
We could have used a Python docstring here and terminated the comment with a period.